hypervisor/include/arch/x86/page.h

```diff
@@ -77,7 +99,7 @@ union pgtable_pages_info {
 
 struct memory_ops {
 	union pgtable_pages_info *info;
-	bool large_page_enabled;
+	bool (*large_page_support)(enum _page_table_level level);
```

Comment:
With the assumption of EPT & PPT uses same page granularity, can we remove this field?